packages/walletconnect-v2/src/index.ts

```diff
@@ -166,15 +158,12 @@ export class WalletConnect extends Connector {
 
   /** {@inheritdoc Connector.deactivate} */
   public async deactivate(): Promise<void> {
-    await this.provider
+    this.provider
```

Comment:
Why did this used to be `await`ed?

Reply:
Guess I carried it over from wc1 that I used as a base where this returned Promise<void>. Doesn't return promise anymore and was very silently flagged by VSCode while refactoring.

Reply:
I don't know, I guess my mistake overlooked in another PR. None of these methods return a Promise.